sematic/db/migrate.py

```diff
@@ -53,14 +53,14 @@ def _is_migration_file(file_name: str) -> bool:
 def _get_current_versions() -> List[str]:
     with db().get_engine().connect() as conn:
         conn.execute(
-            (
+            text(
```

Comment:
raw `execute(some_str)` is no longer supported in SQLAlchemy 2.0; you have to wrap in `text`.